launcher/cloudbuild.yaml

```diff
@@ -21,7 +21,7 @@ steps:
     if [ -z ${base_image} ]
     then
       echo "getting the latest COS image"
-      base_image=$(gcloud compute images describe-from-family ${BASE_IMAGE_FAMILY} --project cos-cloud | grep name | cut -d ' ' -f 2)
+      base_image=$(gcloud compute images describe-from-family ${BASE_IMAGE_FAMILY} --project confidential-vm-images | grep name | cut -d ' ' -f 2)
```

Comment:
Why are we changing the project?

Reply:
COS TDX_capable images are not released in regular cos-cloud project